website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Podman Desktop Release v0.10.0
```

Comment:
- The title could be more exciting, we want to give people the desire to read the content and learn more about what's in.
- IMO, the patch version number is meaningless in our context. 
- Why not trying to callout some of the most exciting features of the release?

Reply:
https://github.com/containers/podman-desktop/pull/943/commits/ed51bbcba5b04ecdae8a6406aabbd2040153097a